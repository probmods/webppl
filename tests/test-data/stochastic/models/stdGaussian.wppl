var model = function() {
  return gaussian();
};
