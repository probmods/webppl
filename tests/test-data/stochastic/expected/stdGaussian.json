{
  "mean": 0,
  "std": 1
}
